app/Models/Playlist/PlaylistItems.php

```diff
@@ -616,11 +616,10 @@ public function fileset()
     public function complete()
     {
         $user = Auth::user();
-        $completed_item = PlaylistItemsComplete::firstOrNew([
+        PlaylistItemsComplete::updateOrCreate([
```

Comment:
upsert?

Reply:
@bradflood , I have updated it. See: https://github.com/faithcomesbyhearing/dbp/pull/937/commits/50131cd2aaa5ca4a38819adf4bd5dbf8f8ecf668#diff-c84de5575df7ebdb3adb9ef2fb5c18db841e439334ba7bfb788e0cf41fa65a0bR624